src/shared/config.js

```diff
@@ -1,5 +1,17 @@
 export const apiEndpoint = process.env.API_ENDPOINT || 'http://localhost:8000/api';
 export const env = process.env.NODE_ENV;
-export const couchDbLocal = 'http://127.0.0.1:5984';
-export const couchDbRemote = `http://${process.env.DB_USERNAME}:${process.env.DB_PASSWORD}@ec2-54-229-76-71.eu-west-1.compute.amazonaws.com:5984/`;
+
+
+export const couchDb = {
```

Comment:
this is great - just make sure to change rest of the app code not to break now after this change